Formula/xcinfo.rb

```diff
@@ -1,17 +1,11 @@
 class Xcinfo < Formula
   desc "Tool to get information about and install available Xcode versions"
   homepage "https://github.com/xcodereleases/xcinfo"
-  url "https://github.com/xcodereleases/xcinfo/archive/0.6.0.tar.gz"
-  sha256 "f1078911c5bff7d123d9e4020685669cae2e1bc74ed95dfefbb4da54b8132353"
+  url "https://github.com/xcodereleases/xcinfo/archive/0.7.0.tar.gz"
+  sha256 "7d5c34c7c4deda28b101c747d89ca6535fd1d50ea26c957e50d18ebeea3da8bb"
   license "MIT"
 
-  bottle do
```

Comment:
There is no need to remove the bottle block

Reply:
OK, I re-added it and did a force push.